### PR TITLE
Raiders of the Capital Wasteland

### PR DESCRIPTION
**Pull request by Superfly-Johnson**
Added Raiders of the Capital Wasteland to the masterlist.

### CODE REVIEW

**Review by pStyl3 — CHANGES_REQUESTED** [review]
@Superfly-Johnson Please rebase, so that only your own commit is within the PR.

File: masterlist.yaml [hunk 1]
```diff
@@ -4260,6 +4260,21 @@ plugins:
         util: '[FO3Edit v4.0.3h (Hotfix 1)](https://www.nexusmods.com/fallout3/mods/637)'
         itm: 18
 
+  - name: 'Evergreen.esm'
+    url: [ 'https://www.nexusmods.com/fallout3/mods/25835' ]
```
Comment (pStyl3) [inline masterlist.yaml]: We add trailing slashes `/` at the end of links & ideally we add the link name as well. So please write the link like this:
```yaml
    url:
      - link: 'https://www.nexusmods.com/fallout3/mods/25835/'
        name: 'Raiders of the Capital Wasteland'
```

File: masterlist.yaml [hunk 2]
```diff
@@ -4260,6 +4260,21 @@ plugins:
...
+    dirty:
+      - <<: *quickClean
+        crc: 0x95821B05
+        util: '[FO3Edit v4.0.4](https://www.nexusmods.com/fallout3/mods/637)'
+        itm: 5
+    clean:
+      - crc: 0xEEBA6BAB
+        util: 'FO3Edit v4.0.4'
```
Comment (pStyl3) [inline masterlist.yaml]: Please remove this part, as we don't add `clean` info, when the plugin is `dirty`.

**Review by pStyl3 — CHANGES_REQUESTED** [review]

File: masterlist.yaml (hunk 2 above)
Comment (Superfly-Johnson) [reply masterlist.yaml]: Done.

**Review by pStyl3 — APPROVED** [review]

**Comment by pStyl3** [comment]
Thanks!

**Pull request merged**